### PR TITLE
Feature/rerouting

### PR DESCRIPTION
**Pull request by CeeyarRbt**
Fixed issues with the rerouting

### CODE REVIEW

File: chatterbox/package.json.orig [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+  "name": "chatterbox",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "bootstrap": "^4.0.0-beta.3",
+    "history": "^4.7.2",
+<<<<<<< HEAD
```
Comment (KimberleighBeyl) [inline chatterbox/package.json.orig]: remove

File: chatterbox/public/index.html.orig [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+<<<<<<< HEAD
```
Comment (KimberleighBeyl) [inline chatterbox/public/index.html.orig]: remove

File: chatterbox/public/manifest.json.orig [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+<<<<<<< HEAD
```
Comment (KimberleighBeyl) [inline chatterbox/public/manifest.json.orig]: this file should not be merged

File: chatterbox/src/Auth/Register/index.js [hunk 4]
```diff
@@ -1,28 +1,68 @@
...
-	    </Button>
-      </div>
-    );
-  }
+    constructor(props) {
+        super(props);
+    }
+    handleSkip() {
+        this.props.history.push('/');
+        alert('TO-DO: redirect to main page');
```
Comment (KimberleighBeyl) [inline chatterbox/src/Auth/Register/index.js]: remove

File: chatterbox/src/Auth/index.js.orig [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
```
Comment (KimberleighBeyl) [inline chatterbox/src/Auth/index.js.orig]: this file should not be here

**Review by KimberleighBeyl — CHANGES_REQUESTED** [review]
Please remove all .orig files and fix your conflicts

File: chatterbox/.gitignore [hunk 6]
```diff
@@ -19,3 +19,8 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# vscode
+.vscode
+package-lock.json
+package.json
```
Comment (KimberleighBeyl) [inline chatterbox/.gitignore]: should not be in gitignore

Comment (CeeyarRbt) [reply chatterbox/.gitignore]: removed them

File: chatterbox/src/Auth/Register/index.js (hunk 4 above)
Comment (CeeyarRbt) [reply chatterbox/src/Auth/Register/index.js]: removed

File: chatterbox/public/manifest.json.orig (hunk 3 above)
Comment (CeeyarRbt) [reply chatterbox/public/manifest.json.orig]: removed .org files

**Comment by CeeyarRbt** [comment]
Already created a new one,

**Pull request closed without merging**